Create test_post_request function

boushrabettir opened the pull request:
- Imported the `json` module to facilitate validation of data in JSON format.
- Created function to assess whether the provided data conforms to JSON format.
- Within the following three functions, implemented a simple conditional using the function created to evaluate incoming POST requests, ensuring their content adheres to valid JSON standards.

The issue can be viewed here #13

Pradyuman7 reviewed (commented):
Can you also link the issue in this PR please?

boushrabettir commented:
> Can you also link the issue in this PR please?

Yes the issue is #13.

boushrabettir commented:
> Is the build failure expected? And, can you also fix the conflicts in this PR

For every PR I've seen, the build failure is present. Could it be possible to create a new issue for it?

Pradyuman7 reviewed (changes requested):
Doesn't the issue mean that you have to implement this functionality for the server and not just for tests? I think the tests are meant to be written as part of the PR to show that the added functionality works correctly

wrussell1999 commented:
Would love to see this resolved before Week 3 @boushrabettir

boushrabettir commented:
> Doesn't the issue mean that you have to implement this functionality for the server and not just for tests? I think the tests are meant to be written as part of the PR to show that the added functionality works correctly

The issue specifically said before it changed that we implement this into `test_pytest.py`. I'll implement it tonight.

boushrabettir commented:
> Doesn't the issue mean that you have to implement this functionality for the server and not just for tests? I think the tests are meant to be written as part of the PR to show that the added functionality works correctly

I've fixed it up right now the functionality, and also added in the last `POST request` functionality, it seems no one did it.

Pradyuman7 commented on app.py:
```diff
@@ -1,146 +1,123 @@
+@app.route('/resume/skill', methods=['GET', 'POST', 'DELETE'])
 def skill():
-    """
+    '''
```
nit: why are we changing these quotes?

boushrabettir replied:
I didn't change them. It was updated from another merged PR. I can go ahead and update them.

boushrabettir replied:
Resolved [here](https://github.com/MLH-Fellowship/orientation-project-python-23.FAL/pull/27/commits/7af1ed34505885c1dff9bf0a630b7c68de392c90)
